tests/unit/helpers.js

```diff
@@ -15,7 +15,7 @@ export const fakeMessageData = {
   message: 'message',
 };
 
-export const EMPTY_SVG = Buffer.from('<svg viewbox="0 0 1 1"></svg>');
+export const EMPTY_SVG = Buffer.from('<svg viewBox="0 0 1 1"></svg>');
```

Comment:
Due to https://github.com/image-size/image-size/issues/355

An alternative solution is what I dropped in [`42897b0` (#3600)](https://github.com/mozilla/addons-linter/pull/3600/commits/42897b03838b7e83c13e4db7af52ec56088c1d27)

Reply:
> Due to [image-size/image-size#355](https://github.com/image-size/image-size/issues/355)

Would be nice to get fixed in the upstream lib, yeah.

Reply:
Done and this test change was dropped